pkg/gopass/secrets/akv_test.go

```diff
@@ -310,3 +341,21 @@ func FuzzParseAKV(f *testing.F) {
 		ParseAKV(in)
 	})
 }
+
+func TestPwWriter(t *testing.T) {
+	a := NewAKV()
+	p := pwWriter{a: a}
+
+	// multi-chunk passwords are supported
+	_, err := p.Write([]byte("foo"))
+	assert.NoError(t, err)
+
+	_, err = p.Write([]byte("bar\n"))
+	assert.NoError(t, err)
+
+	// but anything after the first line is discarded
+	_, err = p.Write([]byte("baz\n"))
+	assert.NoError(t, err)
+
+	assert.Equal(t, "foobar", a.Password())
```

Comment:
Shouldn't the test also test the content of Body?

Reply:
Yes, that's more comprehensive.